prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div {
+      flex-basis: calc(100% / 6);
+    }
```

Comment:
You can remove this rule set as we only want to put `flex-basis` on "flex items" (aka, the children of a "flex container")